README.md

```diff
@@ -18,29 +18,22 @@ checks using [Open Policy Agent (OPA)](https://www.openpolicyagent.org/).
 **[Preflight _agent_](#agent).**
 While this is happening the docs may be inconsistent and the repo structure may
 change.
+[Docs for this component can still be found here](./docs/installation_manual_in_cluster.md).
```

Comment:
```suggestion
[Docs for the check tool can still be found here](./docs/installation_manual_in_cluster.md).
```

Just to clarify as we mentioned two components in the previous sentence.

Reply:
fixed in aff5c9841c73a6736816db01edf4552ed1edf1df